package.json

```diff
@@ -121,6 +122,7 @@
     "eslint-plugin-react": "7.14.2",
     "exports-loader": "0.7.0",
     "file-loader": "4.1.0",
+    "full-icu": "^1.3.0",
```

Comment:
What is this library used for? The package description on npm doesn't explain what ICU is in this context

Reply:
ICU is [Internalization Components for Unicodes](http://userguide.icu-project.org/intro), and so [NodeJS use this](https://nodejs.org/api/intl.html#intl_providing_icu_data_at_runtime) in i18n APIs.

Unicode data has not been included in the default build for a long time because it generally increases the binary size.

NodeJS v13 include full-icu by default. -> https://github.com/nodejs/node/issues/19214

Reply:
Ah, I didn't see that it's just used for testing. That's totally fine to include then